examples/cancel_operation.js

```diff
@@ -0,0 +1,70 @@
+const { DBSQLClient } = require('../');
+
+const client = new DBSQLClient();
+
+const host = '****.databricks.com';
+const path = '/sql/1.0/endpoints/****';
+const token = 'dapi********************************';
+
+async function getQueryResult(operation) {
+  const utils = DBSQLClient.utils;
+
+  await utils.waitUntilReady(operation, false, () => {});
+  console.log('Fetching data...');
+  await utils.fetchAll(operation);
+  await operation.close();
+
+  return utils.getResult(operation).getValue();
+}
+
+async function cancelQuery(operation) {
+  return new Promise((resolve, reject) => {
+    operation
+      .cancel()
+      .then((status) => {
+        if (status.success()) {
+          resolve();
+        } else {
+          reject(new Error(status.getInfo().join('\n')));
+        }
+      })
+      .catch(reject);
+  });
+}
+
+client
+  .connect({ host, path, token })
+  .then(async (client) => {
+    const session = await client.openSession();
+
+    // Execute long-running query
+    console.log('Running query...');
+    const queryOperation = await session.executeStatement(
+      `
+        select * 
+        from some_large_table
+        order by random() + 2 asc
```

Comment:
Does it make sense to use a functional, long-running query instead of a dummy?

```suggestion
        SELECT id 
        FROM RANGE(100000000)
        ORDER BY RANDOM() + 2 asc
```

Reply:
Great idea @susodapop! Thank you 🙂